bigtable-dataflow-import/src/test/java/com/google/cloud/bigtable/dataflowimport/ImportIntegrationTest.java

```diff
@@ -260,12 +259,51 @@ public void testImportSequenceFile_HBase094FormatWithCurrentDeserializer() throw
     }
   }
 
-  private String getCloudTestDataFolder(boolean isHBase094Format) {
-    return cloudTestDataFolder + (isHBase094Format ? HBASE_094_SUBFOLDER : HBASE_CURRENT_SUBFOLDER);
+  /**
+   * Verifies that a Sequence File with Delete Markers may be imported into Cloud Bigtable.
+   *
+   * <p>As of HBase 1.1.2, the delete-all-cells-in-family-by-exact-timestamp marker cannot be
+   * exported to sequence file because the Serializer used by HBase does not recognize this
+   * cell type. Therefore our test data file does not contain this marker.
+   * Ths marker is still tested in unit tests.
+   */
+  @Test
+  public void testImportSequeneFile_fromGcsWithDeleteMarker() throws Exception {
+    final byte[] r1 = Bytes.toBytes("r1");
+    final byte[] r2 = Bytes.toBytes("r2");
+    final byte[] cf1 = Bytes.toBytes("cf1");
+    final byte[] cf2 = Bytes.toBytes("cf2");
+    final byte[] c1 = Bytes.toBytes("c1");
+    final byte[] c2 = Bytes.toBytes("c2");
+    final byte[] val = Bytes.toBytes("1");
+
+    Set<? extends Cell> expected = Sets.newHashSet(
```

Comment:
Hard to tell what we are testing if we only see the expected the values. I understand that we don't have the test data generation logic yet. Maybe describe the input data here?